cfgen/cfgen

```diff
@@ -1471,11 +1476,66 @@ def pool_drives(m0conf: Dict[Oid, Any],
         assert all_unique(drives), \
             'Pool {!r}: some of disk_refs refer to the same disk'.format(
                 pool_desc['name'])
+    return drives
+
+
+# The job of this function is to return list of drives.
+def pool_drives(m0conf: Dict[Oid, Any],
+                pool_desc: Dict[str, Any],
+                cluster_desc: Dict[str, Any]) -> List[Oid]:
+    ptype = pool_type(pool_desc)
+
+    if ptype is PoolT.dix:
+        cas = [(svc_id, m0conf[svc_id].ctrl_id)
+               for svc_id in cluster_svcs(m0conf, SvcT.M0_CST_CAS)]
+        path = '/dev/null'
+        nodes = cluster_desc.get('nodes')
+        if nodes is None:
+            return []
+        for n in nodes:
+            m0servers = n.get('m0_servers')
+            if m0servers is None:
+                return []
+            for m0 in m0servers:
```

Comment:
Here we are iterating over nodes and m0servers, path variable is getting overwritten, and then we are building ConfDrive object outside the for loops, with path not changing, doesn't look right.

Reply:
Addressed